lib/flowViewport/sceneIndex/fvpSelectionSceneIndex.cpp

```diff
@@ -184,7 +184,7 @@ void SelectionSceneIndex::RemoveSelection(const Ufe::Path& appPath)
 
     HdSceneIndexObserver::DirtiedPrimEntries dirtiedPrims;
     for (const auto& primSelection : primSelections) {
-        if (_selection->Remove(primSelection.primPath)) {
+        if (_selection->Remove(primSelection)) {

```

Comment:
Core change : we now only remove the specific selections corresponding to the deselected UFE path, rather than all selections on the prim.